libs/nft-contract/src/deployed/prod.json

```diff
@@ -0,0 +1 @@
+{"contract":"0x2d00"}
```

Comment:
These variables are generated at build time, so, unfortunately, cannot be moved to env.vars